dkms.in

```diff
@@ -767,7 +767,7 @@ check_version_sanity()
         fi
 
         if ((obsolete == 1)); then
-            echo $"" >&2
+            echo "" >&2
```

Comment:
What does the dollar sign even do on these lines? I've never seen an echo done like this before. You technically don't even need the double quotes unless you you are trying to maintain spaces. Although I know that's a common convention so the entire string is the first argument.

Reply:
I think the `echo $""` was there just to follow convention, but all the `echo $"something"` we have inside the dkms main script it's just to allow future translations. I doubt it will ever happen though. I would just remove them all for the sake of readability.

For empty lines there's not need for translation support, so I remove them all.

Reply:
Ah for translations. Got it